src/Okta.Sdk/HttpRequest.cs

```diff
@@ -29,6 +53,21 @@ public class HttpRequest
         /// </value>
         public object Payload { get; set; }
 
+        private string _contentType;
+
+        /// <summary>
+        /// Gets or sets the content type.
+        /// </summary>
+        public string ContentType
+        {
+            get => _contentType;
+            set
+            {
+                _contentType = value;
+                PayloadHandler = Internal.PayloadHandler.ForContentType(_contentType);
```

Comment:
Nit: Let's remove Internal from here and add the corresponding namespace.

Reply:
I can do that if we prefer, however my Visual Studio settings advise against it:
![image](https://user-images.githubusercontent.com/63638027/85295415-bbd87880-b454-11ea-9ed0-397f9cbd3d4e.png)

Is there a stylecop setting that I'm missing?